generator/template/sql_builder_template.go

```diff
@@ -190,8 +222,7 @@ func getSqlBuilderColumnType(columnMetaData metadata.Column) string {
 	case "numrange":
 		return "NumericRange"
 	default:
-		fmt.Println("- [SQL Builder] Unsupported sql column '" + columnMetaData.Name + " " + columnMetaData.DataType.Name + "', using StringColumn instead.")
```

Comment:
Even if we add array types, there still gonna be some unsupported types, so warning message can remain.

Reply:
The warning is handled in the caller

Reply:
Aha, I see. Maybe we an error would be more in go style.

Reply:
How about?

```
// sqlToColumnType maps the type of a SQL column type to a go jet sql builder column. The second return value returns 
// whether the given type is supported.
func sqlToColumnType(typeName string) (string, bool) {
```

Reply:
Yeah, that is fine as well.